loom/common/core/allocator.h

```diff
@@ -272,20 +212,20 @@ struct loom_allocator
 * operators, if you want an array use the templated vector class.
 *
 ************************************************************************/
-#define lmNew(allocator)                new(allocator, __FILE__, __LINE__)
+#define lmNew(allocator)                new(allocator, __FILE__, __LINE__, 1, 2, 3)
```

Comment:
Ah yeah, I was wondering about those, can they have any bad side-effects, e.g. for perf?